spec/lib/curator/parsers/input_parser_spec.rb

```diff
@@ -1,4 +1,4 @@
-# frozen_string_literal: true
+# not using frozen_string_literal here since we're modifying strings
```

Comment:
There is a way around modifying strings with this set using `'somestring'.dup`. This makes an unfrozen copy of the string. We may want to revisit this once we get this closer to releasing a stable version of this